### PR TITLE
chore: sort the frameworks select dropdown in a more logical way

### PR DESCRIPTION
**Pull request by lmiller1990**
- Closes https://cypress-io.atlassian.net/browse/UNIFY-1805

### User facing changelog

Sort the frameworks in a more structured manner on the select framework screen.

### Additional details

Put in alphabetical order.

Open to other ways to order the list, just threw this together since the current list is just in an arbitrary order.

### How has the user experience changed?

image

### PR Tasks

- [x] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

**Review by elevatebart — APPROVED** [review]
I like this new order. Only comment is: If we ever need to change order again in the future, it could be made easier in the code.

**Comment by jennifer-shehane** [comment]
@lmiller1990 Can you create a corresponding Jira issue to track this work? 

I would prefer just alphabetical as an aside. If this screenshot reflects the current state, it still looks illogical to me.

**Comment by lmiller1990** [comment]
@jennifer-shehane done, also just changed to alphabetical order, much more simple + added tests.

File: packages/launchpad/src/setup/EnvironmentSetup.vue [hunk 1]
```diff
@@ -74,6 +73,7 @@ fragment EnvironmentSetup on Wizard {
     isDetected
     type
     category
+    codeGenFramework
```
Comment (ZachJW34) [inline packages/launchpad/src/setup/EnvironmentSetup.vue:76]: Now that we are sorting alphabetically, this is unused. Looks like we can revert the changes to the `CodeGenFrameworkEnum` and `FrameworkCategoryEnum`

**Comment by emilyrohrbough** [comment]
@lmiller1990 which screenshot is the source of truth? Looks like neither screenshot is in alphabetical order, so I feel like i'm missing something.

File: packages/launchpad/src/setup/EnvironmentSetup.vue (hunk 1 above)
Comment (lmiller1990) [reply packages/launchpad/src/setup/EnvironmentSetup.vue:76]: good piont, done

**Comment by lmiller1990** [comment]
@emilyrohrbough oops, I've updated the screenshot.
@ZachJW34 reverted the changed you pointed out we don't need anymore.

thanks!

**Review by emilyrohrbough — APPROVED** [review]

**Review by ZachJW34 — APPROVED** [review]
